app/screens/AppNavigator/screens/Dex/DexConfirmAddLiquidity.tsx

```diff
@@ -171,6 +170,8 @@ export function ConfirmAddLiquidityScreen (props: Props): JSX.Element {
       <SubmitButtonGroup
         isDisabled={isSubmitting || hasPendingJob || hasPendingBroadcastJob}
         label={translate('screens/ConfirmAddLiq', 'ADD')}
+        isSubmitting={isSubmitting || hasPendingJob || hasPendingBroadcastJob}
+        submittingLabel={translate('screens/ConfirmAddLiq', 'Adding Liquidity')}
```

Comment:
Just use ADDING here, to prevent overflow of long texts

Reply:
Will be updating `Removing Liquidity` to `REMOVING` and `Swapping Token` to `SWAPPING`